doc/releases/changelog-dev.md

```diff
@@ -283,6 +283,9 @@ array([False, False])
 * The gradients module no longer needs shot information passed to it explicitly, as the shots are on the tapes.
   [(#4448)](https://github.com/PennyLaneAI/pennylane/pull/4448)
 
+* The private `QuantumScript._prep` list has been removed, and prep operations now go into the `_ops` list.
+  [(#4485)](https://github.com/PennyLaneAI/pennylane/pull/4485)
```

Comment:
I don't think we usually add developer facing changes to the changelog?

Reply:
I kinda figured that although it's "private"/developer-facing, people probably use it because it's the only way to get the list of prep ops. So I put it here because this is the only true change in this PR

Reply:
Fair enough 👍🏼